src/main/java/com/onarandombox/MultiverseCore/world/SimpleMVWorldManager.java

```diff
@@ -103,23 +103,27 @@ public SimpleMVWorldManager(
         this.worlds = new ConcurrentHashMap<String, MVWorld>();
     }
 
+
     /**
      * {@inheritDoc}
      */
     @Override
     public void getDefaultWorldGenerators() {
         this.defaultGens = new HashMap<>();
-        File[] files = server.getWorldContainer().listFiles((file, s) -> s.equalsIgnoreCase("bukkit.yml"));
-        if (files != null && files.length == 1) {
-            FileConfiguration bukkitConfig = YamlConfiguration.loadConfiguration(files[0]);
-            if (bukkitConfig.isConfigurationSection("worlds")) {
-                Set<String> keys = bukkitConfig.getConfigurationSection("worlds").getKeys(false);
-                for (String key : keys) {
-                    defaultGens.put(key, bukkitConfig.getString("worlds." + key + ".generator", ""));
-                }
+
+        File bukkitConfigFile = plugin.getBukkitConfig();
+        if (bukkitConfigFile == null) {
+            Logging.warning("Any Default worldgenerators will not be loaded!");
+            return;
+        }
+
+        FileConfiguration bukkitConfig = YamlConfiguration.loadConfiguration(bukkitConfigFile);
+
+        if (bukkitConfig.isConfigurationSection("worlds")) {
+            Set<String> keys = bukkitConfig.getConfigurationSection("worlds").getKeys(false);
+            for (String key : keys) {
+                defaultGens.put(key, bukkitConfig.getString("worlds." + key + ".generator", ""));
```

Comment:
We might want to wait until this is dealt with in the new world manager...

Reply:
I think its fine,  new world manager wont be touching generators stuff anytime soon